features/support/world.js

```diff
@@ -152,6 +151,30 @@ function ensurePipelineExists(config) {
                     case 'parallel_B2':
                         this.parallel_B2JobId = job.id;
                         break;
+                    case 'simple':
```

Comment:
Where do we use these values?

Reply:
it's used here.
https://github.com/sonic-screwdriver-cd/screwdriver/blob/d3e1e2bef048831705278df0a3dd0235be6e3fe5/features/trigger.feature#L72

Reply:
Could you tell me a more specific place?
I still don't know which code does use these kinds of variables.
https://github.com/screwdriver-cd/screwdriver/pull/2159/files#diff-2bdd2f3b6b7a4cd5baf33e3462df7091R147

Reply:
Sorry, these values are unnecessary.